com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1362,76 +1383,81 @@ private void ConfigureSimulator()
 #if UTP_TRANSPORT_2_0_ABOVE
                 sendQueueCapacity: m_MaxPacketQueueSize,
                 receiveQueueCapacity: m_MaxPacketQueueSize,
-#endif
+#endif //UTP_TRANSPORT_2_0_ABOVE
                 heartbeatTimeoutMS: transport.m_HeartbeatTimeoutMS);
 
             driver = NetworkDriver.Create(m_NetworkSettings);
 
 #if MULTIPLAYER_TOOLS_1_0_0_PRE_7
 #if UTP_TRANSPORT_2_0_ABOVE
             driver.RegisterPipelineStage<NetworkMetricsPipelineStage>(new NetworkMetricsPipelineStage());
-#endif
-#endif
+#endif //UTP_TRANSPORT_2_0_ABOVE
+#endif //MULTIPLAYER_TOOLS_1_0_0_PRE_7
 
 #if UNITY_EDITOR || DEVELOPMENT_BUILD
+#if !UNITY_MP_TOOLS_NETSIM_IMPLEMENTATION_ENABLED
```

Comment:
This + the one on 1434 feels a bit too messy. Consider extracting methods to simplify this stuff a bit and not doing preprocessor surgery on if/else flow.

Reply:
I'll extract those in methods.